lighthouse-core/test/audits/estimated-input-latency-test.js

```diff
@@ -37,8 +37,8 @@ describe('Performance: estimated-input-latency audit', () => {
     const artifacts = generateArtifactsWithTrace({traceEvents: pwaTrace});
     return Audit.audit(artifacts).then(output => {
       assert.equal(output.debugString, undefined);
-      assert.equal(output.rawValue, 17.4);
-      assert.equal(output.displayValue, '17.4ms');
+      assert.equal(output.rawValue, 16.7);
```

Comment:
to be clear, these changes are only due to the shift from speedline.first to FMP. yes?